pkgs/tools/networking/curl/7.15.nix

```diff
@@ -34,7 +36,7 @@ stdenv.mkDerivation rec {
   '';
 
   configureFlags = [
-      "--with-ca-bundle=/etc/ssl/certs/ca-certificates.crt"
+      "--with-ca-bundle=/foobar/ca-certificates.crt"
```

Comment:
Sorry this was a left-over from debugging. Fixed in 6908e75441f5b34dc923a66a8dfbdde0cb7ed9ec